docs/en/interfaces/http.md

```diff
@@ -23,11 +23,13 @@ Web UI can be accessed here: `http://localhost:8123/play`.
 ![Web UI](../images/play.png)
 
 
-In health-check scripts use `GET /ping` request. This handler always returns “Ok.” (with a line feed at the end). Available from version 18.12.13.
+In health-check scripts use `GET /ping` or `GET /replicas_status` request. This handler always returns “Ok.” (with a line feed at the end). Available from version 18.12.13.
```

Comment:
replica_status does a bit more than just returning Ok, and doesn't always return Ok

https://github.com/ClickHouse/ClickHouse/blob/04be5437d94e397c0d379bc648c56950f90f09ef/src/Server/ReplicasStatusHandler.cpp#L25